frontend/portals/TabBar/index.jsx

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import { TAB_BAR_BLACKLIST } from '../../constants';
+
+/**
+ * @param {Node} children children
+ * @returns {JSX}
+ */
+const TabBar = ({ path, children }) => {
```

Comment:
there are much better ways to hide the TabBar:

1. use `configuration.set(TAB_BAR_PATTERNS_BLACK_LIST, list)` https://github.com/shopgate/pwa/blob/e510453fa35774b35c179b73d672401a945f9d56/themes/theme-ios11/components/TabBar/subscriptions.js#L50 

afaik we have the read the current blacklist. add our own routes to it and set it again. but i'm only 99% sure

2. use `TabBar.show()` and `TabBar.hide()`

We should go with 1 here. 2 is just fyi for the future

Reply:
Method 1. worked with just adding out own routes to the current blacklist.